development-tools/warnings.rst

```diff
@@ -0,0 +1,69 @@
+.. warnings:
+
+=================================
```

Comment:
We can skip this overline:
```suggestion
```

The convention is here: https://devguide.python.org/documentation/markup/#sections

We often skip the overlines though.

Reply:
Removed overfine. I had just copied how other pages in development-tools were under/overlining.